travis.sh

```diff
@@ -69,16 +207,78 @@ function runFindbugs() {
   fi
 }
 
+##########################################################
+# Run CPD for the langauge passed, check for failures and
+# move the final result to a separate file.
+#
+# Arguments:
+#   arg1: Language for which CPD is run (Java or Scala)
+#   arg2: Duplicates threshold for the language
+#   arg3: Name of the threshold constant for the language
+#   arg4: List of files changed in the PR
+# Returns:
+#   None
+##########################################################
+function runCPDForLanguage() {
```

Comment:
Duplicate code, can be moved to cpd.sh.

Reply:
The code isnt really duplicate because we are handling changed files here which we do not handle in compile.sh